### ISSUE TITLE
Update link

### ISSUE THREAD

**Issue by vpetersson** [NebraLtd/hm-miner#5]
We're pulling down from the old VPS here:

```
wget -O /opt/miner/releases/0.1.0/sys.config https://hmapi.nebra.com/docker.config
```

in `start_miner.sh`.

That needs to be:
* A documented why we pull down a `docker.config` file
* Moved elsewhere

**Comment by shawaj** [comment]
Think this was fixed in https://github.com/NebraLtd/hm-miner/commit/6f5f5cb4a9bd29ce3d1ef69f151f64471f084f16

**Comment by shawaj** [comment]
Although guess some documentation still would not go amiss

**Comment by vpetersson** [comment]
Yeah for sure. We'll document it when we're done with the snapshot'ing tool. It's still a moving target.
